src/test/java/org/testcontainers/containers/TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest.java

```diff
@@ -21,8 +21,7 @@ public class TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest {
                     "Dockerfile",
                     "cartridge/instances_fixedport.yml",
                     "cartridge/topology_fixedport.lua")
-                    .withCopyFileToContainer(MountableFile.forClasspathResource("cartridge"), "/app")
```

Comment:
Let's leave this since on Windows mapping folders may not work. These lines were added for a purpose. We need to check that the new code will work on all platforms

Reply:
Probably we need to use copy on Windows platform in the container code too, and then remove it here

Reply:
Another option is to make copy optional, and pass it to the container setup. That looks like the most universal solution.